sympy/functions/special/hyper.py

```diff
@@ -282,8 +282,7 @@ def _eval_nseries(self, x, n, logx, cdir=0):
         for i in range(n):
             num = Mul(*[RisingFactorial(a, i) for a in ap])
             den = Mul(*[RisingFactorial(b, i) for b in bq])
-            terms.append(((num/den) * (arg**i)) / factorial(i))
-
+            terms.append(((num/den) * (arg**i).nseries()) / factorial(i))
```

Comment:
It is not sufficient to just call `nseries` here without passing any arguments. Also possibly there needs to be some checking here of what nseries returns.